provider/team_resource_test.go

```diff
@@ -135,23 +136,20 @@ func testAccCheckTeamResourceExistsWithAttributes_basic(resourceName string) res
 			return fmt.Errorf("No ID is set")
 		}
 
-		client, err := firehydrant.NewRestClient(os.Getenv("FIREHYDRANT_API_KEY"))
-		if err != nil {
-			return err
-		}
+		client := fhsdk.New(fhsdk.WithSecurity(components.Security{APIKey: os.Getenv("FIREHYDRANT_API_KEY")}))
```

Comment:
This works, but calling `firehydrant.NewRestClient()` and then using `client.Sdk` is more consistent with how we're initializing the client in the provider and might be safer against changes in the future.

Reply:
good callout, will change this to be consistent

Reply:
Oh even better.  Good call.